torch/csrc/Exceptions.h

```diff
@@ -75,12 +75,7 @@ static inline void PyErr_SetString(PyObject* type, const std::string& message) {
   _CATCH_GENERIC_ERROR(LinAlgError, THPException_LinAlgError, retstmnt) \
   _CATCH_GENERIC_ERROR(                                                 \
       OutOfMemoryError, THPException_OutOfMemoryError, retstmnt)        \
-  _CATCH_GENERIC_ERROR(Error, PyExc_RuntimeError, retstmnt)             \
-  catch (torch::PyTorchError & e) {                                     \
-    auto msg = torch::processErrorMsg(e.what());                        \
-    PyErr_SetString(e.python_type(), msg);                              \
-    retstmnt;                                                           \
-  }
+  _CATCH_GENERIC_ERROR(Error, PyExc_RuntimeError, retstmnt)
```

Comment:
You want to add the newly added `AttributeError` to the list above so that it gets properly translated into an AttributeError in the python side.